roles/prometheus/templates/prometheus.yml.j2

```diff
@@ -19,17 +19,26 @@ scrape_configs:
     metrics_path: /prometheus/metrics
     static_configs:
       - targets: ['localhost:9090']
-  - job_name: 'dummy' # it's mandatory
-    file_sd_configs:
-      - files:
-        - /etc/prometheus/file_sd/*.json
+  - job_name: blackbox
+    metrics_path: /probe
+    params:
+      module: [http_prometheus]
+    static_configs:
+      - targets:
+{%- for url in probe_url %}        
+         - {{ url }}
+{% endfor %}
```

Comment:
do we need the `relabel_configs` portion of the config: https://github.com/prometheus/blackbox_exporter#prometheus-configuration

Reply:
Yes it was needed otherwise it complains about hostname. Added it.